scripts/simex

```diff
@@ -809,39 +810,71 @@ queue_daemon_parser.set_defaults(cmd=do_queue_daemon)
 queue_daemon_parser.add_argument('--force', action='store_true')
 
 def do_queue_stop(args):
-	simexpal.queuesock.stop_queue()
+	try:
+		simexpal.queuesock.stop_queue()
+	except FileNotFoundError:
+		print("There is currently no queue daemon running.")
+	except ConnectionRefusedError:
+		print("There is currently a queue daemon running that was not closed properly. Use "
+			"'simex queue interactive --force' or 'simex q daemon --force' to forcefully open a new daemon. "
+			"Alternatively you can delete the socket '{}' manually and start a new daemon.".format(
+			simexpal.base.DEFAULT_SOCKETPATH))
```

Comment:
"was not closed" -> "did not terminate". "open" -> "launch". See do_queue_interactive.